upgrade/upgrade-scripts/common.sh

```diff
@@ -81,16 +81,21 @@ function verify_upgrade_is_allowed() {
 			"($MINIMUM_VERSION)"
 }
 
-function verify_upgrade_in_place_is_allowed() {
+function is_reboot_required_upgrade() {
```

Comment:
can we use consistent terminology between this, and the "verify" function below..

i.e. either settle on "reboot required" or "in-place allowed"

I'm fine with either one.. it just seems like we have two different terms to mean the same thing, so if we could make both use the same terminology that might make things more clear to readers of this code.